randnum.c

```diff
@@ -0,0 +1,47 @@
+//Clarke Littlejohn
```

Comment:
You implemented the assignment correct, but not efficiently. Instead of a while loop, you could have simply written r = rand() % 10 + 1;